joomla/joomla-cms
# com_languages languages view, ordering in searchtools

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_languages content languages view.

Also corrects some code issues and adds Home ordering.

And other small code changes/improvements.
###### Before PR

![com_languages-languages-beforepr](https://cloud.githubusercontent.com/assets/9630530/12962765/bd57b8fa-d03e-11e5-9125-c1b7413652be.png)
###### After PR

![com_languages-languages-afterpr](https://cloud.githubusercontent.com/assets/9630530/12962771/c17f206c-d03e-11e5-8e23-7bb656d1aa31.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Extensions -> Languages -> Content languages
3. Check the ordering select box and the filter are working fine.
